manifests/init.pp

```diff
@@ -102,9 +105,9 @@
   $use_sympa           = false,         # postfix_use_sympa
   $postfix_ensure      = 'present',
   $mailx_ensure        = 'present',
-) inherits postfix::params {
-
+) { include ::postfix::params
```

Comment:
Ah right. I didn't feel it was my place to refactor the whole module to accomodate small change this but maybe there isn't really any other option.  How do you want (me) to proceed here?